packages/nx-go/src/generators/library/generator.ts

```diff
@@ -22,5 +22,11 @@ export default async function (tree: Tree, options: LibraryGeneratorSchema) {
     tags: normalizedOptions.parsedTags,
   })
   addFiles(tree, join(__dirname, 'files'), normalizedOptions)
+
+  if (normalizedOptions.useGoWork) {
+    createGoMod(tree, normalizedOptions)
```

Comment:
Do we need to call `createGoMod` when creating a library?

Reply:
I think I already understand why this is, we need a `go.mod` in all the separate projects of the workspace, is that correct?